pkg/controller/appdefinition/testdata/computeclass/generic-resources/existing.yaml

```diff
@@ -0,0 +1,25 @@
+---
+kind: ClusterComputeClassInstance
+apiVersion: internal.admin.acorn.io/v1
+metadata:
+  name: sample-compute-class
+description: Simple description for a simple ComputeClass
+cpuScaler: 0.25
+memory:
+  min: 1Mi
+  max: 2Mi
+  default: 1Mi
+resources:
+  limits:
+    gpu-vendor.example/example-limit: 1
+  requests:
+    gpu-vendor.example/example-request: 1
```

Comment:
I don't actually think this test is working properly. If you look at the `expected.golden` that got generated, it doesn't actually have these limits in it.

Reply:
right, good catch thanks @tylerslaton 
Do I understand correctly that in this test I have to set in input.yaml the AppInstance expected after the scheduling? Meaning that `status.scheduling.oneimage.requirements` in input.yaml should already contain the gpu information? That makes me wonder if existing.yaml has any effect other than reference.

While the scheduling part is validated a part in the scheduling folder.

Reply:
You are correct in your understanding that `input.yaml` is expected to be the AppInstance after the `scheduling` handler has run. You are also correct in that `existing.yaml` has no effect other than reference. The fix here is to add your `resources` section into the `input.yaml` so that we can validate it actually appears in the underlying Deployment, which is what the `computeclass_test.go` cases are testing for.

Reply:
awesome, thank you. In that case the commits I pushed yesterday are in line with what you just said.